### ISSUE TITLE
Aclaración Tarea 12

### ISSUE THREAD

**Issue by ajahr95** [IIC2343/syllabus#279]
Si bien la tarea 12 será elegida por el cuerpo de ayudantes, ¿Esta podrá ser de cualquier tipo? Es decir, ¿Práctica, de programación o teórica?

**Comment by Geeermy** [comment]
Sí, salvo las teóricas cuya pauta esté arriba, por razones obvias.
